lib/migration.js

```diff
@@ -423,7 +432,7 @@ function mixinMigration(MySQL, mysql) {
         fkSQL = self.getForeignKeySQL(model, newFks);
     }
     if (fkSQL && fkSQL.length) {
-      self.applySqlChanges(model, fkSQL, function(err, result) {
+      self.applySqlChanges(model, [fkSQL.join(', ')], function(err, result) {
```

Comment:
I agree that `self.execute(self.getAlterStatement(model, fkSQL), function(err, result) {` would do what we were trying to. Other connectors use `applySqlChanges ` function and we should keep them consistent.